content/articles/a-3d-object-detection-solution-for-everyday-objects/index.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /a-3d-object-detection-solution-for-everyday-objects/
+title: A 3D Object Detection Solution for Everyday Objects
+description: This tutorial will cover how to perform 3D object detection using the mediapipe library and python and draw 3D bounding box around the objects.
+author: lilian-tonia
+date: 2021-12-16T00:00:00-17:20
+topics: [Machine Learning]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/a-3d-object-detection-solution-for-everyday-objects/hero.png
+    alt: A 3D Object Detection Solution for Everyday Objects Hero Image
+---
+In this tutorial, we will perform 3D object detection using the mediapipe library and python. We will be drawing a 3D bounding box around an object instead of the common 2D bounding boxes that we are used to.
+<!--more-->
+
+### Prerequisites
+To follow along with this tutorial, you need to:
+- Be familiar with machine learning modelling.
+- Be familiar with the python programming language.
+- Install either Jupyter notebook or Google colab.
+> To follow along with this tutorial, please use Jupyter Notebook.
+
+### Table of contents
+- [Why 3D detection is important](#why-3d-detection-is-important)
+- [What is the Objectron](#what-is-the-objectron)
+- [How they obtained real-world 3D training data](#how-they-obtained-real-world-3d-training-data)
+- [Detect and track 3D objects using this model](#detect-and-track-3d-objects-using-this-model)
+- [Wrapping up](#wrapping-up)
+- [Further reading](#further-reading)
+
+### Why 3D detection is important
+Over the years, object detection research has solely focused on performing 2D object detections. We've seen this with the likes of RCNN, Fast RCNN, SSD, and Masked RCNN. In the real-world, we have 3D objects. Because of this, it would be better if we had 3D bounding boxes to bound objects detected in the real world, rather than the commonly used 2D detections. 
+
+3D object detection is important as it would enable us capture object’s sizes, orientation, and position in the world. As a result, we would be able to use these 3D detections in real-world applications such as Augmented Reality (AR), self-driving cars, and robotics which perceive the world the same way we do as humans.
```

Comment:
```suggestion
3D object detection is important as it would enable us to capture objects’ sizes, orientation, and position in the world. As a result, we would be able to use these 3D detection in real-world applications such as Augmented Reality (AR), self-driving cars, and robotics which perceive the world the same way we do as humans.
```